src/Store/ExtensionClientStoreConnector.re

```diff
@@ -121,13 +121,28 @@ let start = (extensions, setup: Core.Setup.t) => {
       ExtHostClient.pump(extHostClient)
     );
 
+  let activatedFileTypes: Hashtbl.t(string, bool) = Hashtbl.create(16);
+
+  let activateFileType = (fileType: option(string)) =>
+    switch (fileType) {
+    | None => ()
+    | Some(ft) =>
+      switch (Hashtbl.find_opt(activatedFileTypes, ft)) {
+      | None =>
+        ExtHostClient.activateByEvent("onLanguage:" ++ ft, extHostClient);
+        Hashtbl.add(activatedFileTypes, ft, true);
+      | Some(_) => ()
+      }
+    };
```

Comment:
`Option.bind` could flatten this a bit

Reply:
Good point! I used `Option.iter` and added an `Option.iter_none`